great_expectations/datasource/fluent/fluent_base_model.py

```diff
@@ -64,7 +61,7 @@ class Config:
 
     @classmethod
     def parse_yaml(cls: Type[_Self], f: Union[pathlib.Path, str]) -> _Self:
-        loaded = yaml.load(f)
+        loaded = yaml.load(f)  # type: ignore[arg-type]
```

Comment:
I'm somewhat inclined to remove the return type for `YAMLHandler.load`, it's not very helpful and causes a lot of issues.